src/frontends/pytorch/include/openvino/frontend/pytorch/frontend.hpp

```diff
@@ -60,7 +60,7 @@ class PYTORCH_API FrontEnd : public ov::frontend::FrontEnd {
     bool supported_impl(const std::vector<ov::Any>& variants) const override;
     ov::frontend::InputModel::Ptr load_impl(const std::vector<ov::Any>& variants) const override;
 
-    std::map<std::string, PytorchCreatorFunction> m_op_translators;
+    std::map<std::string, CreatorFunction> m_op_translators;
```

Comment:
`unordered_map`?

Reply:
We have map in all frontends, we could change to unordered_map, but this is out of scope of this PR

Reply:
np